virtualizarr/xarray.py

```diff
@@ -142,6 +162,30 @@ def open_virtual_dataset(
                 if name in loadable_variables
             }
 
+            for name in cftime_variables:
+                import cftime
+
+                var = ds[name]
+                calendar = var.attrs.get(
+                    "calendar", var.encoding.get("calendar", "standard")
+                )
+                units = var.attrs.get("units", var.encoding["units"])
```

Comment:
Can we abstract this out into a function? That would also help make it clearer what the Zarr Codec equivalent would be.

Reply:
yes!